src/demo/index.ts

```diff
@@ -102,15 +97,24 @@ export function startBpmnVisualization(config: BpmnVisualizationDemoConfiguratio
 
   const parameters = new URLSearchParams(window.location.search);
 
-  fitOnLoad = parameters.get('fitOnLoad') == 'true';
-  log(`Configure 'fit on load' to ${fitOnLoad}`);
+  log('Configure load options');
+  const parameterFitType = parameters.get('fitType');
+  if (parameterFitType) {
+    options.fitType = FitType[parameterFitType as keyof typeof FitType];
```

Comment:
:question: add console log for consistency with other options?

Reply:
Done line 106